test/libsolidity/syntaxTests/parsing/interface_basic.sol

```diff
@@ -2,4 +2,4 @@ interface Interface {
 	function f() public;
 }
 // ----
-// Warning: (23-43): Functions in interfaces should be declared external.
+// TypeError: (23-43): Functions in interfaces must be declared external.
```

Comment:
I think this is supposed to pass as well (and merely test that interfaces go through the parser).